languages/translations/de.json

```diff
@@ -84,23 +84,34 @@
         },
         "continue": {
             "default": {
-                "default": "{modifier} weiterfahren",
-                "name": "{modifier} weiterfahren auf {way_name}",
-                "destination": "{modifier} weiterfahren Richtung {destination}"
+                "default": "Turn {modifier}",
+                "name": "Turn {modifier} to stay on {way_name}",
+                "destination": "Turn {modifier} towards {destination}",
```

Comment:
Before we merge this, let’s go into Transifex and accept any `100% match` suggestions so that perfectly acceptable translations don’t get overridden by the original English versions.

Reply:
@1ec5 - I'm not sure how to do that